cocos/asset/asset-manager/pack-manager.ts

```diff
@@ -107,7 +117,8 @@ export class PackManager {
                 }
             } else {
                 err = new Error('unmatched type pack!');
-                out = null;
+                onComplete(err, null);
```

Comment:
Problem: if don't do this, the `out` should be `null | Record<...>` which else causes many assertions in above.